### PR TITLE
docs: fix instructions of lasagna master

### PR DESCRIPTION
**Pull request by fabiodinatale**
The aim of this PR is to try to clarify the instructions of Task 3.

### CODE REVIEW

File: exercises/concept/lasagna-master/.docs/instructions.md [hunk 1]
```diff
@@ -49,8 +49,7 @@ It was amazing and had something special to it.
 The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
 Now you want to add that secret ingredient to your recipe as well.
 
-Write a function `addSecretIngredient` that accepts two `string` vectors of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list of your own recipe.
+Write a function `addSecretIngredient` that accepts a reference to a `string` of vectors representing the ingredient list of your own recipe and a `string` of vectors that is the list your friend sent you.
```
Comment (vaeng) [inline exercises/concept/lasagna-master/.docs/instructions.md:52]: I prefer the two sentence version. It should be a `vector of strings`.
How about we change line 53 to `The first parameter is a reference to the list your friend ...`?

Comment (fabiodinatale) [reply exercises/concept/lasagna-master/.docs/instructions.md]: Thanks @vaeng.
With [this commit](8db1be52d957df04c07cde78f5ee97681d1d8d83) I tried to address your suggestion.

File: exercises/concept/lasagna-master/.docs/instructions.md [hunk 2]
```diff
@@ -49,8 +49,8 @@ It was amazing and had something special to it.
 The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
 Now you want to add that secret ingredient to your recipe as well.
 
-Write a function `addSecretIngredient` that accepts two `string` vectors of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list of your own recipe.
+Write a function `addSecretIngredient` that accepts two `vector of strings` as parameters.
+The first one is a reference to the ingredient list of your own recipe, while the second one is the `vector of strings` of the ingredient list your friend sent you.
```
Comment (vaeng) [inline exercises/concept/lasagna-master/.docs/instructions.md:53]: That is a good clarification. We should let the users decide what to do. Both parameters can be references. For performance they both should be, and the second one should be _const_. 

I think Siebenschläfer and I had the cpp terms in _italics_ and parts from the actual code in `blocks`.

What do you think about that?

```suggestion
The first parameter is a _reference_ to the ingredients of your own recipe, and the second _vector_ of strings has the ingredients your friend sent you.
```

Comment (fabiodinatale) [reply exercises/concept/lasagna-master/.docs/instructions.md]: With [this commit](4c09b8a7d3e11976157ada59ad8a58681a5ccad5), I added in the note the possibility of using a reference also for the friend's list.
The fact that it has not to be modified was already specified there.

I also accepted the suggestion for the style of the instructions.

**Comment by fabiodinatale** [comment]
Can we proceed merging the *PR* @vaeng?

**Review by vaeng — APPROVED** [review]

File: exercises/concept/lasagna-master/.docs/instructions.md [hunk 3]
```diff
@@ -49,12 +49,12 @@ It was amazing and had something special to it.
 The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
 Now you want to add that secret ingredient to your recipe as well.
 
-Write a function `addSecretIngredient` that accepts two `string` vectors of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list of your own recipe.
+Write a function `addSecretIngredient` that accepts two `vector of strings` as parameters.
+The first parameter is a _reference_ to the ingredients of your own recipe, and the second _vector_ of strings has the ingredients your friend sent you.
```
Comment (iHiD) [inline exercises/concept/lasagna-master/.docs/instructions.md:53]:
```suggestion
The first parameter is a _reference_ to the ingredients of your own recipe, and the second has the ingredients your friend sent you.
```

Comment (iHiD) [reply exercises/concept/lasagna-master/.docs/instructions.md:53]: (Partially just suggesting this to kick the CI if it's accepted, but also I don't think we need this bit :))

Comment (vaeng) [reply exercises/concept/lasagna-master/.docs/instructions.md:53]: CI kick did not work :(

**Comment by vaeng** [comment]
Do you have any ideas @ErikSchierboom

**Comment by ErikSchierboom** [comment]
I've closed and reopened, that often works

**Comment by vaeng** [comment]
Thanks @ErikSchierboom

**Comment by fabiodinatale** [comment]
Thanks @vaeng!

**Pull request merged**